env/types.go

```diff
@@ -212,6 +220,14 @@ func ParseBool(value, name string) (string, error) {
 	return "", fmt.Errorf(text.FormatFieldInvalidValue, value, name, "Boolean")
 }
 
+// The var QUIET and ON_ERROR_STOP treat an empty value as "on"
```

Comment:
Since you mentioned idiomatic code, it's idiomatic to start comments with the function name:
```suggestion
// ParseBoolEmptyIsOn works like ParseBool but treats empty values as "on"
```